src/olympia/files/tests/test_models.py

```diff
@@ -899,7 +899,8 @@ def setUp(self):
         self.version = Version.objects.create(addon=self.addon)
 
     def upload(self, name):
-        if os.path.splitext(name)[-1] not in ['.xml', '.xpi', '.jar', '.zip']:
+        extension = os.path.splitext(name)[-1]
+        if extension not in ['.xml', '.xpi', '.jar', '.zip', '.crx']:
```

Comment:
I just realised we should be importing the constant from [`models.py`](https://github.com/mozilla/addons-server/pull/3009/files#diff-ecb8df04bee7aac11123f3729e23cfafR37) here, having the extra `.json` shouldn't hurt anything (and is technically correct, I imagine, anyway).